### PR TITLE
AggregationDataStore: Change JPA & Hibernate Stores to Implement Brid…

### PR DESCRIPTION
**Pull request by QubitPi**
Issue https://github.com/yahoo/elide/issues/821

### CODE REVIEW

File: elide-datastore/elide-datastore-hibernate5/src/main/java/com/yahoo/elide/datastores/hibernate5/HibernateTransaction.java [hunk 1]
```diff
@@ -242,6 +246,31 @@ public Object getRelation(
         return val;
     }
 
+    @Override
+    public Object bridgeableLoadObject(
```
Comment (aklish) [inline elide-datastore/elide-datastore-hibernate5/src/main/java/com/yahoo/elide/datastores/hibernate5/HibernateTransaction.java]: We shouldn't need to define these.

File: elide-datastore/elide-datastore-multiplex/src/main/java/com/yahoo/elide/datastores/multiplex/BridgeableTransaction.java [hunk 2]
```diff
@@ -54,12 +55,27 @@
...
                                 Serializable lookupId,
                                 Optional<FilterExpression> filterExpression,
-                                RequestScope scope);
+                                RequestScope scope) {
+        EntityDictionary entityDictionary = scope.getDictionary();
+
+        if (lookupId == null) {
+            // according to contract, lookupId can be null
+            // in this case, we compute the lookupId
+            Object parentInstance = muxTx.loadObject(
```
Comment (aklish) [inline elide-datastore/elide-datastore-multiplex/src/main/java/com/yahoo/elide/datastores/multiplex/BridgeableTransaction.java]: The parentInstance is already loaded here (parent parameter in the function).

File: elide-datastore/elide-datastore-multiplex/src/main/java/com/yahoo/elide/datastores/multiplex/BridgeableTransaction.java [hunk 3]
```diff
@@ -54,12 +55,27 @@
...
+                    entityDictionary.getEntityClass(parent.getClass().getName()),
+                    entityDictionary.getId(parent),
+                    filterExpression, scope
+            );
+            lookupId = entityDictionary.getId(entityDictionary.getField(parentInstance, relationName));
```
Comment (aklish) [inline elide-datastore/elide-datastore-multiplex/src/main/java/com/yahoo/elide/datastores/multiplex/BridgeableTransaction.java]: I understand what you are trying to do here, but I think the whole contract for store bridging is busted.  I don't know if there is a general way to get the ID from the parent.  It seems like the other store (the one we bridged from would have to supply it somehow).

File: elide-core/src/main/java/com/yahoo/elide/core/EntityDictionary.java [hunk 4]
```diff
@@ -991,6 +991,14 @@ public String getId(Object value) {
         }
     }
 
+    public AccessibleObject getField(Object value, String fieldName) {
```
Comment (hellohanchen) [inline elide-core/src/main/java/com/yahoo/elide/core/EntityDictionary.java]: Can this be renamed as `getFieldValue`?

**Pull request closed without merging**